docs/EPs/003-ReusableComponents.md

```diff
@@ -14,7 +14,7 @@ through the hierarchy in the form of `props`.  All very functional.
 
 Except there are some problems:
   - it can be a PITA to pass every little bit of data through many, many layers. 
-    Manual and time consuming.
+    Manual and time-consuming.
```

Comment:
Its not clear what the rule is here... https://english.stackexchange.com/a/498296

But I'll accept it anyway 😄

Reply:
I was not sure as well (not native English speaker). For all the fixes, I double checked on Google each time.